components/compliance-service/inspec/cli.go

```diff
@@ -47,6 +48,21 @@ func isTimeoutSane(timeout time.Duration, max time.Duration) error {
 	return nil
 }
 
+func AcceptInspecLicense(home string) error {
+	licenseDir := home + "/.chef/accepted_licenses"
+	if _, err := os.Stat(licenseDir + "/inspec"); os.IsNotExist(err) {
+		err := os.MkdirAll(licenseDir, os.ModePerm)
+		if err != nil {
+			return errors.Wrap(err, "InSpec license mkdir")
+		}
+		_, err = os.Create(licenseDir + "/inspec")
+		if err != nil {
+			return errors.Wrap(err, "InSpec license create")
+		}
+	}
+	return nil
+}
```

Comment:
We know what `TmpDir` is when the app starts up and don't allow changing it afterwards (right?), so we could do this check once while the app is starting. That feels simpler code-wise and also feels like a better user experience in the case where something goes wrong (ie, the service fails to start rather than every one of these function calls failing later).

Reply:
Not needed anymore as AcceptInspecLicense is only called when the service starts now. Thank you!